test/Generators/Microsoft.Gen.MetadataExtractor/GoldenReports/MeterDimensionsAttributedWithXmlDescriptions_LogMethod.json

```
@@ -1,71 +1,76 @@
                         {
-                            "Name": "user",
-                            "Type": "string",
-                            "File": "src-0.cs",
-                            "Line": "14",
-                            "Classifications": [
+                            "Name": "LogHello",
+                            "Parameters": [
                                 {
-                                    "Name": "C2",
-                                    "Notes": "Note 3"
+                                    "Name": "user",
+                                    "Type": "string",
+                                    "File": "src-0.cs",
+                                    "Line": "14",
+                                    "Classifications": [
+                                        {
+                                            "Name": "C2",
+                                            "Notes": "Note 3"
+                                        }
+                                    ]
+                                },
+                                {
+                                    "Name": "port",
+                                    "Type": "int",
+                                    "File": "src-0.cs",
+                                    "Line": "14"
                                 }
                             ]
-                        },
-                        {
-                            "Name": "port",
-                            "Type": "int",
-                            "File": "src-0.cs",
-                            "Line": "14"
                         }
                     ]
                 }
             ]
-        }
-    ]
-} , "MetricReport": [
- {
-  "TestClasses":
-  [
-    {
-     "MetricName": "DescribedDimensionCounter",
-     "MetricDescription": "(Missing Summary)",
-     "InstrumentName": "Counter",
-     "Dimensions": {
-      "Dimension1": "Dimension1 description.",
-      "Dim1": ""
-      }
-    },
-    {
-     "MetricName": "DescribedDimensionHistogram",
-     "MetricDescription": "(Missing Summary)",
-     "InstrumentName": "Histogram",
-     "Dimensions": {
-      "Dimension2": "Dimension2 description.",
-      "DimensionDefinedInMetricClass": "DimensionDefinedInMetricClass description."
-      }
-    },
-    {
-     "MetricName": "MyStrongTypeMetricWithDescription",
-     "MetricDescription": "(Missing Summary)",
-     "InstrumentName": "Counter",
-     "Dimensions": {
-      "AnotherDimension": "Gets or sets anotherDimension.",
-      "MetricEnum": "Gets or sets MetricEnum.",
-      "MetricEnum2": "Gets or sets MetricEnum2.",
-      "Dim2": "Gets or sets Dim2.",
-      "SomeDim": "Gets or sets SomeDim.",
-      "Dim4Struct": "Gets or sets Dim4Struct.",
-      "Dim5Struct": "Gets or sets Dim5Struct."
-      }
-    }
-  ]
- }
-] }
\ No newline at end of file
+        },
+    "MetricReport": 
+        [
+            {
+                "TestClasses":
+                [
```

For example here, arrays should start the bracket on the same line as the property, and containing objects indented only one level

```suggestion
    "MetricReport": [
        {
            "TestClasses": [
```

---

the reason for this is that all the generators start with an empty line.
the `MetaDataGenerator `utilizes the  `MetricsReportsGenerator` which will start with a empty line by default. 
if you look at `ComplianceReports` it will be the same.

---

i can for example introduce a new flag to decide whither to keep the newline or not